cosette/core.py

```diff
@@ -31,13 +31,13 @@
 # %% ../00_core.ipynb 5
 empty = inspect.Parameter.empty
 
-# %% ../00_core.ipynb 7
-models = 'o1-preview', 'o1-mini', 'gpt-4o', 'gpt-4-turbo', 'gpt-4', 'gpt-4-32k', 'gpt-3.5-turbo', 'gpt-3.5-turbo-instruct'
+# %% ../00_core.ipynb 6
+o1_models = 'o1-preview', 'o1-mini'
```

Comment:
I changed `text_only_models` to `o1_models` because I wanted to reallocate the term `text_only` to refer to the message format.